### PR TITLE
RBTA-22-login-functionality-registration-kashina

### PR DESCRIPTION
**Pull request by MarinaKashina**
created test login registration click in the login.js

### CODE REVIEW

**Comment by artsenius** [comment]
You don't have to create a new PR everytime you need to change something. Just update, commit and push it. It will automatically go to already existing PR.

File: test/specs/reg/login.js [hunk 1]
```diff
@@ -19,6 +19,14 @@ describe('Login', function () {
       input.addValue('j'.repeat(maxLenght + 1));
       let actual = input.getValue().length;
       assert.equal(actual, maxLenght);
+    });
+
+    it('Registration', function () {
+      browser.url('/');
+      $('#registration').click();
+      let email_confirm = $('#email_confirm');
```
Comment (artsenius) [inline test/specs/reg/login.js]: Everything looks fine except two hardcoded values - "#registration" and "#email_confirm".
So, now the next step is to have them in locators and use as variables.

Comment (MarinaKashina) [reply test/specs/reg/login.js]: I was changed to the test

File: test/selectors/login.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default {
+    registration: '#registration',
+    email_confirm: '#email_confirm'
```
Comment (artsenius) [inline test/selectors/login.js]: We agreed to use camel case, for example emailConfirm, but NOT email_confirm

File: test/specs/reg/login.js [hunk 3]
```diff
@@ -19,6 +20,13 @@ describe('Login', function () {
       input.addValue('j'.repeat(maxLenght + 1));
       let actual = input.getValue().length;
       assert.equal(actual, maxLenght);
+    });
+
+    it('Registration', function () {
+      let reg = $(sel.registration).click();
+      let email_c = $(sel.email_confirm);
```
Comment (artsenius) [inline test/specs/reg/login.js]: camel case please

**Comment by artsenius** [comment]
Plus you need to resolve conflicts
I can help you tomorrow

File: test/selectors/login.js (hunk 2 above)
Comment (MarinaKashina) [reply test/selectors/login.js]: Thanks for this comment. I added the camel case

File: test/specs/reg/login.js (hunk 3 above)
Comment (MarinaKashina) [reply test/specs/reg/login.js]: I added the camel case

File: test/specs/reg/login.js [hunk 4]
```diff
@@ -19,6 +20,13 @@ describe('Login', function () {
       input.addValue('j'.repeat(maxLenght + 1));
       let actual = input.getValue().length;
       assert.equal(actual, maxLenght);
+    });
+
+    it('Registration', function () {
+      let reg = $(sel.registration).click();
```
Comment (artsenius) [inline test/specs/reg/login.js]: no sense to declare variable here. just click

File: test/specs/reg/login.js [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import { assert } from 'chai';
+import sel from '../../selectors/login';
 let maxLenght = 45;
```
Comment (artsenius) [inline test/specs/reg/login.js]: store this variable in expected data. not here

File: test/selectors/client.js [hunk 6]
```diff
@@ -3,4 +3,5 @@ export default {
   header: '.custom-header',
   footer: '.custom-footer',
   app: '.site-content'
+
```
Comment (artsenius) [inline test/selectors/client.js]: remove extra line

**Pull request merged**